torch/csrc/inductor/aoti_torch/generated/c_shim_cpu.h

```diff
@@ -0,0 +1,112 @@
+
+// WARNING: THIS FILE IS AUTOGENERATED BY torchgen. DO NOT MODIFY BY HAND.
```

Comment:
Need to say how to update this file, no? Because it doesn't get updated as part of the build process?